crates/amalthea/src/kernel.rs

```diff
@@ -104,7 +104,6 @@ pub fn connect(
     )?;
     let shell_port = port_finalize(&shell_socket, connection_file.shell_port)?;
 
-    let shell_clone = shell_handler.clone();
     let iopub_tx_clone = iopub_tx.clone();
     let lsp_handler_clone = lsp_handler.clone();
     let dap_handler_clone = dap_handler.clone();
```

Comment:
While we are here - these don't need to be clones either!

Reply:
I wish the compiler would let us know.